### ISSUE TITLE
Zero log2-fold change but credible change being identified?

### ISSUE THREAD

**Issue by learning-MD** [theislab/scCODA#66]
Thanks for this great tool, which has been very helpful in several projects! However, I've run into an issue that I'm not sure how to interpret. My understanding of the results interpretation was that we typically should expect a log2-fold change to be something other than 0 if there is a credible change in cluster composition: https://sccoda.readthedocs.io/en/latest/getting_started.html

However, when I run the following code, I see 0 for all log2-fold changes:
```
model_all = mod.CompositionalAnalysis(data_scanpy, formula="Cond", reference_cell_type="Platelets")
all_results = model_all.sample_hmc()
all_results.summary()
```
![image](https://user-images.githubusercontent.com/88126859/189684943-bb7a8d34-635b-49b8-a26f-f0093e6f1f57.png)

When checking credible changes, however, several populations are listed as TRUE:

`print(all_results.credible_effects())`

![image](https://user-images.githubusercontent.com/88126859/189685016-0f3df172-4b83-4410-a04c-7a9abee83f05.png)

Is this expected behavior? Or is there an issue I'm missing? I'm not sure how to interpret these results. When I use "automatic" as my reference cell type (the "pDCs" group), only one cell population is described as having a credible change but still with a log2-fold change of 0. Visually, at least, that cluster appears to be smaller on a stacked barplot grouped by condition.

**Comment by johannesostner** [comment]
Hi @learning-MD !
This behaviour is unexpected, and your understanding of the results is correct. From your screenshots I am unable to figure out what the problem is here though.

Could you please add 
- copies or screenshots of `all_results.effect_df` and `all_results.intercept_df` or the output of `all_results.summary_extended()`
- some information about your data (how many samples and cell types do you have?) and the version of scCODA that you use

Thank you!

**Comment by learning-MD** [comment]
Thanks for the quick response! I have a total of 40 samples (roughly 20 in the disease and 20 in the control). I'm using scCODA version 0.1.3 - maybe an update is what's needed?

Below is the output of `all_results.summary_extended()`

![image](https://user-images.githubusercontent.com/88126859/189696552-c695834b-0af0-43e7-8eb4-d5a8631ff034.png)
![image](https://user-images.githubusercontent.com/88126859/189696664-01fbad9f-effa-4844-b1ea-12989b6b47c0.png)
![image](https://user-images.githubusercontent.com/88126859/189696818-86d99455-c24e-4f4f-8488-4f77a3ae5b0b.png)
![image](https://user-images.githubusercontent.com/88126859/189696888-31816c38-727b-4ef2-8f4d-8ac7b94fe6d7.png)

Below is the output of `all_results.effect_df`
![image](https://user-images.githubusercontent.com/88126859/189697232-c464d26f-80f2-4132-b131-b18bc282d5c4.png)

And below is the output of `all_results.intercept_df`
![image](https://user-images.githubusercontent.com/88126859/189697385-f67ebc08-52e2-4ba6-8134-14993cd9bb7d.png)

**Comment by learning-MD** [comment]
As an update, I updated scCODA to the newest version (0.1.8) and this issue seems to be resolved. Both the log2-fold change and credible change results are consistent.

**Comment by johannesostner** [comment]
Thanks for the update!
There was a bug in earlier versions of `credible_effects` (see #37), which was fixed in version 0.1.4, so this was likely the issue.
Closing this for now.
